pkg/server/config.go

```diff
@@ -185,6 +177,45 @@ func NewConfig(opts *kcpserveroptions.CompletedOptions) (*Config, error) {
 
 	c.GenericConfig.RequestInfoResolver = requestinfo.NewFactory() // must be set here early to avoid a crash in the EnableMultiCluster roundtrip wrapper
 
+	// a note on the identities in a multi-shard environment
```

Comment:
This comment lacks the "what are we doing here" preamble to explain what identities we're talking about and why we need them